pkg/test/resourcefixture/testdata/basic/bigqueryreservation/v1alpha1/bigqueryreservationreservation/failovertononfailoverreservation/_generated_object_failovertononfailoverreservation.golden.yaml

```diff
@@ -12,14 +12,15 @@ metadata:
 spec:
   autoscale:
     maxSlots: 50
-  concurrency: 0
+  concurrency: 10
   edition: ENTERPRISE_PLUS
+  failover:
```

Comment:
The updated golden object should not include a non-nil `failover.secondaryLocation`, this means the controller implementation regarding this field is probably not correct.